### PR TITLE
[infra/onert] Build and install luci library

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit introduces build and install luci library to makefile and gbs spec file.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: https://github.com/Samsung/ONE/issues/11117
Draft: https://github.com/Samsung/ONE/pull/11050 https://github.com/Samsung/ONE/pull/11298

### CODE REVIEW

File: packaging/nnfw.spec [hunk 1]
```diff
@@ -254,6 +282,13 @@ install -m 0644 ./tests/scripts/build_path.txt %{buildroot}%{test_install_dir}/t
 %endif # coverage_build
 %endif # test_build
 
+%if %{odc_build} == 1
+mkdir -p %{buildroot}%{_libdir}/nnfw/odc
+install -m 644 %{overlay_path}/lib/libluci*.so %{buildroot}%{_libdir}/nnfw/odc
+install -m 644 %{overlay_path}/lib/libloco*.so %{buildroot}%{_libdir}/nnfw/odc
+#install -m 644 build/out/lib/nnfw/odc/*.so %{buildroot}%{_libdir}/nnfw/odc
```
Comment (hseok-oh) [inline packaging/nnfw.spec:289]: It will be enabled when `libonert_odc.so` is ready.

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-x64-release

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-cross-aarch64-release

**Review by glistening — APPROVED** [review]
LGTM :+1:

**Review by ragmani — APPROVED** [review]
LGTM :+1:

**Pull request merged**